### PR TITLE
Pico calibration sim

### PR DESCRIPTION
**Pull request by zonca**

### CODE REVIEW

File: calibration_sim/pico_scanning.par [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+--hwprpm
+0.0
+--baseline
+60.0
+--nside
+1024
+--obs
+24.0
+--input_pysm_model
+a1,d1,f1,s1
```
Comment (zonca) [inline calibration_sim/pico_scanning.par]: @ziotom78, are you fine with the suggestion of @jdborrill to have: 

> unlensed scalar CMB (r=0) with an intrinsic dipole

? can you take care of producing it?

File: calibration_sim/pico_scanning.par [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+--precperiod
+14400.0
+--precangle
+22.0
+--hwprpm
+0.0
+--baseline
+60.0
+--nside
+1024
```
Comment (zonca) [inline calibration_sim/pico_scanning.par]: @jdborrill is 1024 fine? or could we do 512 to lower memory requirements?

**Comment by zonca** [comment]
@jdborrill @ziotom78, I am preparing the configuration files to run PICO calibration sims, still work in progress but please take a look at them. As suggested by @ziotom78, I am setting up band 9 - 90GHz.

File: calibration_sim/pico_scanning.par (hunk 1 above)
Comment (ziotom78) [reply calibration_sim/pico_scanning.par]: Sure, I'll do it immediately.

Comment (ziotom78) [reply calibration_sim/pico_scanning.par]: Done, CAMB output is in `/scratch1/scratchdirs/tomasi/PICO_simulations/cmb_sky`. What is of interest for this simulation is the CMB sky map (with NSIDE=1024), saved in `camb_20960065_map.fits`.

Comment (zonca) [reply calibration_sim/pico_scanning.par]: thanks @ziotom78  can you please make a pull request on my `pico_calibration_sim` branch and add a subfolder `cmb_sky` with the CAMB configuration (& scripts if useful)?

Comment (ziotom78) [reply calibration_sim/pico_scanning.par]: @zonca, I used the web interface at https://lambda.gsfc.nasa.gov/toolbox/tb_camb_form.cfm, and I did not prepare any configuration file. Which files should I add to the repository? The text file with all the alms? The NSIDE=1024 map is perhaps too large to be uploaded.

Comment (zonca) [reply calibration_sim/pico_scanning.par]: ok, the alms are fine. too bad that the form doesn't allow to download a text configuration file that can be used to run the CAMB executable from the terminal...

File: calibration_sim/pico_scanning.par (hunk 2 above)
Comment (zonca) [reply calibration_sim/pico_scanning.par]: @jdborrill agreed 512 is fine see 
c4cdd98

**Comment by zonca** [comment]
@ziotom78 got the CMB from Julian, it is at `/project/projectdirs/pico/data/sky_00/ffp10_unlensed_scl_cmb_000_alm.fits`
However it has a primordial dipole, I suggest I remove it, are you fine with this or you'd rather leave it in?

**Comment by ziotom78** [comment]
I would prefer to remove it, otherwise the exercise gets more complicated. We'll think about how to deal with data which include it later.

**Comment by zonca** [comment]
thanks @ziotom78, I have everything I need, I'll let you know once I have the output files.

**Comment by zonca** [comment]
input CMB at NSIDE 512 available at `/global/cscratch1/sd/zonca/pico/cal_sims/pico_cmb_nodip.fits` I zeroed the dipole alm, it still has a residual dipole of 10 microK.

**Comment by zonca** [comment]
memory consumption was low so I ran 6 months, results are very strange, I need  to understand what is going on. 
pico_band9_6months

**Comment by zonca** [comment]
fixed a couple of issues, I have tested each component separately on 60 days, then here the total 185 days map:

pico_185

@ziotom78 timelines are in `/global/cscratch1/sd/zonca/pico/cal_sims/185days/out_000`

**Pull request merged**